### PR TITLE
influxdb v0.9.0

### PR DESCRIPTION
**Pull request by gunnaraasen**

### CODE REVIEW

File: Library/Formula/influxdb.rb [hunk 1]
```diff
@@ -4,208 +4,126 @@
 class Influxdb < Formula
   desc "Time series, events, and metrics database"
   homepage "http://influxdb.com"
-  url "https://s3.amazonaws.com/get.influxdb.org/influxdb-0.8.8.src.tar.gz"
-  sha1 "a5686d0374ec5ab616e335e9c5fb1aeacd17fb00"
+  url "https://github.com/influxdb/influxdb/archive/v0.9.0.tar.gz"
+  sha256 "c7cc869754d7bfb9374b0a16a9b91073d9588adedbd7bfda1d5709e22d3a3d75"
+  version "0.9.0"
```
Comment (DomT4) [inline Library/Formula/influxdb.rb]: Don't need this. The URL is simple enough for `brew` to determine a version from.

File: Library/Formula/influxdb.rb [hunk 2]
```diff
@@ -4,208 +4,126 @@
...
 
-  bottle do
```
Comment (DomT4) [inline Library/Formula/influxdb.rb]: Please leave the bottle block as is - We'll update it automatically on pull.

**Comment by gunnaraasen** [comment]
@DomT4 made some updates. I didn't realize the bottle was auto generated. I've added it back in.

**Comment by 9mm** [comment]
exciting

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/ea4b5171dc4c0457d75fe42521ab272df0647c22. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Comment by 9mm** [comment]
How do I install 0.8.8 with brew? Unfortunately 0.9.0 is lacking a very large amount of features that I need, and I'm not sure how to install the older version, considering it's not in homebrew/versions or homebrew/graveyard

**Comment by DomT4** [comment]
> considering it's not in homebrew/versions

File a PR for it in `Versions`.

**Pull request closed without merging**